docs/source/config.rst

```diff
@@ -73,23 +75,100 @@ There are 8 different sections :
     :enabled: Check local(temporary) logging is enabled
     :PATH: Location of file for local(temporary) logging
 
-If no file is specified, following json will be used as default:
+If no file is specified, following YAML will be used as default:
 
 .. code-block:: python
 
-    {'DATA': {'db_config': '/opt/tanner/db/db_config.json', 'dorks': '/opt/tanner/data/dorks.pickle',
-              'user_dorks': '/opt/tanner/data/user_dorks.pickle',
-     'TANNER': {'host': '0.0.0.0', 'port': 8090},
-     'WEB': {'host': '0.0.0.0', 'port': 8091},
-     'API': {'host': '0.0.0.0', 'port': 8092},
-     'PHPOX': {'host': '0.0.0.0', 'port': 8088},
-     'REDIS': {'host': 'localhost', 'port': 6379, 'poolsize': 80, 'timeout': 1},
-     'EMULATORS': {'root_dir': '/opt/tanner'},
-     'EMULATOR_ENABLED': {'sqli': 'True', 'rfi': 'True', 'lfi': 'True', 'xss': 'True', 'cmd_exec': 'True'},
-     'SQLI': {'type':'SQLITE', 'db_name': 'tanner_db', 'host':'localhost', 'user':'root', 'password':'user_pass'},
-     'DOCKER': {'host_image': 'busybox:latest'},
-     'LOGGER': {'log_file': '/opt/tanner/tanner.log'},
-     'MONGO': {'enabled': 'False', 'URI': 'mongodb://localhost'},
-     'LOCALLOG': {'enabled': 'False', 'PATH': '/tmp/tanner_report.json'},
-     'CLEANLOG': {'enabled': 'False'}
-     }
+  DATA:
```

Comment:
You can use comments to describe a little bit sections :)
I find this example quite good :)
https://opennmt.net/OpenNMT-tf/configuration.html